packages/currency-utils/src/currencyUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { Connection, PublicKey } from "@solana/web3.js";
+import { TOKEN_PROGRAM_ID, AccountLayout } from "@solana/spl-token";
+
+export const getBalance = async ({
+  pubKey,
+  mint,
+  connection,
+}: {
+  connection: Connection;
+  pubKey: PublicKey;
+  mint: string;
+}) => {
+  const tokenAccounts = await connection.getTokenAccountsByOwner(pubKey, {
+    programId: TOKEN_PROGRAM_ID,
+  });
+
+  const tokenAcct = tokenAccounts.value.find((ta) => {
+    const accountData = AccountLayout.decode(ta.account.data);
+    return accountData.mint.toBase58() === mint;
+  });
+  if (!tokenAcct) return;
+
+  return Number(AccountLayout.decode(tokenAcct.account.data).amount);
+};
+
+export const getSolBalance = async ({
```

Comment:
Is this needed? Method already exists on connection

Reply:
Definitely not needed, I'll remove